sample/Sample/Sample.csproj

```diff
@@ -16,6 +16,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.1.1" />
     <PackageReference Include="Serilog.Sinks.Async" Version="1.0.1" />
+    <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" />
     <PackageReference Include="Serilog.Sinks.Literate" Version="2.0.0" />
```

Comment:
I think the package Serilog.Sinks.Literate is no longer required, because the sample app only use the Console Sink